src/store/reducers/appReducer.js

```diff
@@ -121,7 +122,29 @@ const initialState = {
   isDefaultLogo: true,
   azureMapsKey: '',
   deviceGroupFlyoutIsOpen: false,
-  timeInterval: 'PT1H'
+  timeInterval: 'PT1H',
+
+  //TODO: Get this from the server. This is just hardcoded test data for now.
+  userPermissions: [
```

Comment:
Will do. But will leave as individual items in an array instead of using the "Object.values(permissions)".  With each listed in the array here, it is easier to test each one individually by commenting one but not the others.  When the list is retrieved from the server, this will be a non-issue.